coregrind/m_execontext.c

```diff
@@ -7,7 +7,7 @@
    This file is part of Valgrind, a dynamic binary instrumentation
    framework.
 
-   Copyright (C) 2000-2015 Julian Seward 
+   Copyright (C) 2000-2015 Julian Seward
```

Comment:
Please don't touch unrelated lines when modifying things outside of pmemcheck directory. Such changes may be hard to rebase when updating pmemcheck to the next version.